cloud-controller-manager/do/cloud.go

```diff
@@ -57,10 +57,14 @@ const (
 	publicAccessFirewallTagsEnv string = "PUBLIC_ACCESS_FIREWALL_TAGS"
 	regionEnv                   string = "REGION"
 	doAPIRateLimitQPSEnv        string = "DO_API_RATE_LIMIT_QPS"
+	doIPAddrFamiliesEnv         string = "DO_IP_ADDR_FAMILIES"
 )
 
 var version string
 
+var ipFamilyOpts = []string{"", "ipv4", "ipv6", "ipv4,ipv6", "ipv6,ipv4"}
```

Comment:
Having to specify all orderings seems a bit unclean and doesn't allow for white spaces to be used. How about introducing a custom string type with two constants instead, one for each family?

We can then also reference the family in both the bootstrap and the business logic.

Reply:
Sure, makes sense, fixed it.